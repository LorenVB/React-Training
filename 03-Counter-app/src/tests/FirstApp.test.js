import React from "react";
import { shallow } from "enzyme";
const { default: FirstApp } = require("../FirstApp");

describe("Testing FirstApp", () => {
            // test("should return Hola, soy Goku", () => {
            //         const saludo = "Hola, soy Goku";
            //         const { getByText } = render( < FirstApp saludo = { saludo }
            //             />);

            //             expect(getByText(saludo)).toBeInTheDocument();
            //         });

            test("return <FirstApp /> correctly", () => {
                    const saludo = "Hola, Soy Goku";
                    const wrapper = shallow( < FirstApp saludo = { saludo }
                        / > );

                        expect(wrapper).toMatchSnapshot();
                    });
            });