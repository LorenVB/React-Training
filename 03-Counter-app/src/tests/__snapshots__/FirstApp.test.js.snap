// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing FirstApp return <FirstApp /> correctly 1`] = `
<Fragment>
  <h1>
     
    Hola, Soy Goku
    !! 
  </h1>
   
   
   
   
   
  <p>
     
    Soy un subtitulo
     
  </p>
   
   
</Fragment>
`;
